src/test/java/javax0/license3j/io/TestKeyPairReader.java

```diff
@@ -13,7 +13,7 @@
     @Test
     @DisplayName("Can read public key from BINARY file")
     void canReadPublicKeyFromBinaryFile() throws IOException, InvalidKeySpecException, NoSuchAlgorithmException {
-        try (final var sut = new KeyPairReader(this.getClass().getResource(TestKeyPairWriter.PUBLIC_KEY_BINARY).getFile())) {
+        try (final var sut = new KeyPairReader(this.getClass().getResource(TestKeyPairWriter.PUBLIC_KEY_BINARY).getFile().replaceAll("%20"," "))) {
```

Comment:
This is a cheap and dirty and incomplete approach to the issue.

I refactored these two tests and I will push that to the next master commit.